### PR TITLE
doc: fix view config doc

### PR DESCRIPTION
**Pull request by CodeFalling**
- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

fix view config in quickstart

### CODE REVIEW

File: docs/source/en/intro/quickstart.md [hunk 1]
```diff
@@ -174,11 +174,18 @@ exports.nunjucks = {
...
-  mapping: {
-    '.tpl': 'nunjucks',
-  },
+module.exports = appInfo => {
+  const config = {};
+  config.keys = appInfo.name + '...';
+
+  // add config
+  config.view = {
+      defaultViewEngine: 'nunjucks',
```
Comment (atian25) [inline docs/source/en/intro/quickstart.md]: 2 spaces

File: docs/source/en/intro/quickstart.md [hunk 2]
```diff
@@ -174,11 +174,18 @@ exports.nunjucks = {
 
 ```js
 // config/config.default.js
-exports.view = {
```
Comment (fengmk2) [inline docs/source/en/intro/quickstart.md]: What's the problem in here?

**Comment by CodeFalling** [comment]
Default `config/config.default.js` in current version is：

```
module.exports = appInfo => {
  const config = {};

  // should change to your own
  config.keys = appInfo.name + '...';

  // add your config here
  return config;
};
```

`exports.view = {...}` in quickstart is confusing. Users would just paste it after origin file and nothing works.

**Comment by popomore** [comment]
其实我觉得脚手架里 keys 不一定要使用 name。

**Comment by atian25** [comment]
@popomore 骨架里面生成的时候顺手加上的，免得重复

**Pull request merged**